core/translate/select.rs

```diff
@@ -511,7 +524,22 @@ fn translate_tables_begin(program: &mut ProgramBuilder, select: &mut Select) ->
     let processed_where = process_where(program, select)?;
 
     for loop_info in &select.loops {
-        translate_table_open_loop(program, select, loop_info, &processed_where)?;
+        // if there is a left join and there is a condition on the join that is always false,
+        // every row in the outer table will be emitted with nulls for the right table
+        let early_terminate_label = if let Some(ljbk) = &loop_info.left_join_bookkeeping {
```

Comment:
s/ljbk/left/ for readability maybe...

Reply:
Renamed ljbk to left_join and left_join_bookkeeping to left_join_maybe